.github/workflows/build.yml

```diff
@@ -0,0 +1,13 @@
+name: Build and test
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build with testing
+      run: ./gradlew --stacktrace test
```

Comment:
I'm not really sure about Github actions, but is there a way to generate / view test report in there?

Reply:
At least shown in the test log.
https://github.com/embulk/embulk-input-sftp/runs/442969143?check_suite_focus=true

Reply:
That log only shows `test` phase has been executed and also failure test cases if any instead of the overall reports of test suite. But it seems Github action does not have that option and also we did not use in TravisCI properly.